lib/src/room.dart

```diff
@@ -51,16 +51,32 @@ abstract class Room implements Listenable {
   Stream<Message> get messageStream;
 
   /// Joins the room with the provided local audio and video stream.
-  ///
-  /// If the room should be joined with the audio and/or video muted, disable the corresponding track(s) before calling
-  /// [join].
   Future<void> join(MediaStream localStream);
 
-  /// Mutes or un-mutes the local audio stream.
-  void setAudioMuted(bool muted);
+  /// Whether the local audio is muted.
+  bool get isAudioMuted;
```

Comment:
Since the `Room` now manages its own mute states, we expose getters to retrieve the mute states: `isAudioMuted`, `isVideoMuted` and `isPresentationMuted`.